packages/transformers/vue/src/VueTransformer.js

```diff
@@ -63,10 +63,24 @@ export default (new Transformer({
       });
     }
 
+    const descriptor = parsed.descriptor;
+    let id = hashObject({
+      filePath: asset.filePath,
+    }).slice(-6);
+
     return {
       type: 'vue',
       version: '3.0.0',
-      program: parsed.descriptor,
+      program: {
+        ...descriptor,
+        script:
+          !!descriptor.script || !!descriptor.scriptSetup
```

Comment:
Could this be rewritten as `... != null || ... != null`? I'd prefer that over `!!`

Reply:
fixed~